### ISSUE TITLE
Can not delete folders in version 6.7.0

### ISSUE THREAD

**Issue by rubanooo** [pimcore/pimcore#6945]
# Bug Report

I updated PimCore to the newest version 6.7.0 today and tried to delete a folder. This issue is reproducable on the demo site.

### Actual behavior

Deleting a folder gives you the following error:
`Call to undefined method getClassName in class Pimcore\Model\DataObject\Folder`

### Steps to reproduce

Create a folder or select an existing folder and try to delete it by opening the right mouse button menu.

### Trace

```json
 [{
 	"file": "\/var\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/DataObject\/AbstractObject.php",
 	"line": 1350,
 	"function": "__call",
 	"class": "Pimcore\\Model\\AbstractModel",
 	"type": "-\u003E",
 	"args": ["getClassName", []]
 }, {
 	"file": "\/var\/www\/pimcore\/vendor\/pimcore\/pimcore\/models\/DataObject\/AbstractObject.php",
 	"line": 361,
 	"function": "__call",
 	"class": "Pimcore\\Model\\DataObject\\AbstractObject",
 	"type": "-\u003E",
 	"args": ["getClassName", []]
 }, {
 	"file": "\/var\/www\/pimcore\/vendor\/pimcore\/pimcore\/bundles\/AdminBundle\/Controller\/Admin\/ElementControllerBase.php",
 	"line": 160,
 	"function": "getList",
 	"class": "Pimcore\\Model\\DataObject\\AbstractObject",
 	"type": "::",
 	"args": [{
 		"unpublished": true
 	}]
 }, {
 	"file": "\/var\/www\/pimcore\/vendor\/pimcore\/pimcore\/bundles\/AdminBundle\/Controller\/Admin\/DataObject\/DataObjectController.php",
 	"line": 92,
 	"function": "deleteInfoAction",
 	"class": "Pimcore\\Bundle\\AdminBundle\\Controller\\Admin\\ElementControllerBase",
 	"type": "-\u003E",
 	"args": [{
 		"attributes": {},
 		"request": {},
 		"query": {},
 		"server": {},
 		"files": {},
 		"cookies": {},
 		"headers": {}
 	}]
 }, {
 	"file": "\/var\/www\/pimcore\/vendor\/symfony\/symfony\/src\/Symfony\/Component\/HttpKernel\/HttpKernel.php",
 	"line": 158,
 	"function": "deleteInfoAction",
 	"class": "Pimcore\\Bundle\\AdminBundle\\Controller\\Admin\\DataObject\\DataObjectController",
 	"type": "-\u003E",
 	"args": [{
 		"attributes": {},
 		"request": {},
 		"query": {},
 		"server": {},
 		"files": {},
 		"cookies": {},
 		"headers": {}
 	}]
 }, {
 	"file": "\/var\/www\/pimcore\/vendor\/symfony\/symfony\/src\/Symfony\/Component\/HttpKernel\/HttpKernel.php",
 	"line": 80,
 	"function": "handleRaw",
 	"class": "Symfony\\Component\\HttpKernel\\HttpKernel",
 	"type": "-\u003E",
 	"args": [{
 		"attributes": {},
 		"request": {},
 		"query": {},
 		"server": {},
 		"files": {},
 		"cookies": {},
 		"headers": {}
 	}, 1]
 }, {
 	"file": "\/var\/www\/pimcore\/vendor\/symfony\/symfony\/src\/Symfony\/Component\/HttpKernel\/Kernel.php",
 	"line": 201,
 	"function": "handle",
 	"class": "Symfony\\Component\\HttpKernel\\HttpKernel",
 	"type": "-\u003E",
 	"args": [{
 		"attributes": {},
 		"request": {},
 		"query": {},
 		"server": {},
 		"files": {},
 		"cookies": {},
 		"headers": {}
 	}, 1, true]
 }, {
 	"file": "\/var\/www\/pimcore\/web\/app.php",
 	"line": 36,
 	"function": "handle",
 	"class": "Symfony\\Component\\HttpKernel\\Kernel",
 	"type": "-\u003E",
 	"args": [{
 		"attributes": {},
 		"request": {},
 		"query": {},
 		"server": {},
 		"files": {},
 		"cookies": {},
 		"headers": {}
 	}]
 }]
```

**Comment by dvesh3** [comment]
regression of https://github.com/pimcore/pimcore/pull/6689

**Comment by brusch** [comment]
Fixed by #6947
